CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Grant method was added for `*_ready_buffer` spaces.
```

Comment:
```suggestion
- Grant method was added for `*_ready_buffer` spaces (#239).
```

Reply:
Isn't it #237?

I thought that we are providing the issue nubmer.

Reply:
Oh, yes, sorry 237.

> I thought that we are providing the issue nubmer.

We don't have a general rule here, but we provide issue numbers in this repository.

https://github.com/tarantool/queue/blob/3bfd1e2519695da32816f19953182118d9b63626/CHANGELOG.md?plain=1#L30

Reply:
Done.